DSA/graph.cpp

```diff
@@ -0,0 +1,275 @@
+#include <bits/stdc++.h>
+using namespace std;
+
+// printing function
+
+void print(string message)
+{
+    cout << endl
+         << message;
+}
+
+// Graph Class
+
+class Graph
+{
+public:
+    // Using adjency list for making this graph
+
+    map<int, vector<int>> adj;
```

Comment:
You don't have to use built in DS . Change it

Reply:
changes done created graph with adjacency matrix